infrastructure/sandbox.tfvars

```diff
@@ -1,3 +1,5 @@
-idam_url = "https://idam-web-public-idam-saat.service.core-compute-idam-saat.internal"
-idam_api_url = "http://idam-api-idam-saat.service.core-compute-idam-saat.internal/"
+infrastructure_env = "preprod"
+node_environment = "sandbox"
+idam_url = "https://idam-web-public-idam-sandbox.service.core-compute-idam-sandbox.internal"
+idam_api_url = "http://idam-api-idam-sandbox.service.core-compute-idam-sandbox.internal/"
 idam_enable_stub = "false"
```

Comment:
@chrisdavecm , I don't think this is used anymore in saat. Actually, I renamed that file to saat in our services and they deployed successfully in SAAT.